client/components/inline-support-link/use-support-doc-data.ts

```diff
@@ -36,12 +49,21 @@ const useSupportDocData = ( {
 		blogId: 0, // support.wordpress.com is the default blog used for support links
 	} );
 
+	const site = useSelector( getSelectedSite );
+	const isJetpack = useSelector( ( state ) => isJetpackSite( state, site?.ID ) );
```

Comment:
We need to exclude Atomic sites here. See https://github.com/Automattic/wp-calypso/pull/104211#discussion_r2156789363